.github/workflows/linux-eic-shell.yml

```
@@ -781,6 +781,36 @@ jobs:
           python3 ${{ github.workspace }}/epic-capybara/bara.py ref/rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4eic.root rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4eic.root
           echo "::endgroup::"
 
+  trigger-container:
+    runs-on: ubuntu-latest
+    needs:
+    - eicrecon-gun
+    steps:
+    - uses: eic/trigger-gitlab-ci@v2
+      id: trigger
+      with:
+        url: https://eicweb.phy.anl.gov
+        project_id: 290
+        token: ${{ secrets.EICWEB_CONTAINER_TRIGGER }}
+        ref_name: master
+        variables: |
+          GITHUB_REPOSITORY=${{ github.repository }}
+          GITHUB_SHA=${{ github.event.pull_request.head.sha || github.sha }}
+          GITHUB_PR=${{ github.event.pull_request.number }}
+          VERSION="eicrecon-${GITHUB_SHA::8}"
```

Doesn't seem like we can use variables that we are still setting in the same block...
```suggestion
          VERSION="eicrecon-${{ github.event.pull_request.head.sha || github.sha }}"
```